lib/instance.js

```diff
@@ -68,7 +66,7 @@ module.exports = function instance(input, done, progress) {
       try {
         require(module);
       } catch (err) {
-        throw new Error(err);
```

Comment:
I don't know enough about the entire codebase to know whether we really want to throw here, or just log like in every other case. If we need to just throw then we can delete the try/catch altogether.

Reply:
interesting that we are catching/throwing ... already seems like the try/catch is extraneous.